zerver/lib/bugdown/__init__.py

```diff
@@ -1173,6 +1174,15 @@ def run(self, lines):
                 if re.search(match_re, content):
                     current_message.alert_words.add(word)
 
+            for command in slash_commands:
+                escaped = re.escape(("/" + command).lower())
+                match_re = re.compile(u'(?:%s)%s(?:%s)' %
```

Comment:
Just double checking: does this properly handle the case where one slash command is a prefix of another?  For example, if the message contains `/say`, will the slash commands `/s` and `/sa` both get matched?